<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menus</class>
 <widget class="QWidget" name="Menus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="filename_label">
     <property name="text">
      <string>Filename</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tab_widget">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="interpolation_tab">
      <attribute name="title">
       <string>Interpolation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="interpolation_number_of_points">
         <property name="text">
          <string>Number of points</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="number_of_points_spin_box">
         <property name="stepType">
          <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="draw_cubic_spline_button">
         <property name="text">
          <string>Draw interpolation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="interpolation_clear_button">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="interpolation_clear_all_button">
         <property name="text">
          <string>Clear all</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="group_box">
         <property name="title">
          <string notr="true"/>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QDateTimeEdit" name="date_time_interpolation_date_time_edit"/>
          </item>
          <item>
           <widget class="QLabel" name="interpolated_value_label">
            <property name="text">
             <string>Interpolated value</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="interpolation_vertical_spacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="approximation_tab">
      <attribute name="title">
       <string>Approximation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="approximation_number_of_points_label">
         <property name="text">
          <string>Number of points</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="number_of_approximation_points_spin_box">
         <property name="stepType">
          <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="approximation_number_of_predict_days_label">
         <property name="text">
          <string>How many days to predict</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="extends_of_days_spin_box">
         <property name="maximum">
          <number>14</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="approximation_polynomial_degree_label">
         <property name="text">
          <string>Polynomial degree</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="polynomial_power_spin_box">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="draw_approximation_button">
         <property name="text">
          <string>Draw approximation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear_approximation_button">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="approximation_clear_all_button">
         <property name="text">
          <string>Clear all</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QDateEdit" name="date_time_approximate_date_time_edit"/>
          </item>
          <item>
           <widget class="QLabel" name="approximated_value_label">
            <property name="text">
             <string>Approximated value</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="approximation_vertical_spacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>178</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
